app/src/main/res/layout-land/state_fragment.xml

```diff
@@ -115,19 +115,20 @@
 
     <FrameLayout
       android:layout_width="match_parent"
-      android:layout_height="wrap_content"
+      android:layout_height="124dp"
       android:layout_gravity="bottom"
-      android:layout_marginStart="28dp"
       android:layout_marginTop="8dp"
-      android:layout_marginEnd="28dp"
-      android:layout_marginBottom="16dp"
       android:elevation="8dp"
       android:translationZ="4dp">
```

Comment:
Does this have any effect on the confetti being rendered?

Reply:
No from what I can tell, though this should actually be set for the text view -- will move it

Reply:
This actually did hide the confetti so I ended up removing elevation, I couldn't see a visible UI difference

Reply:
Doesn't the elevation create a small shadow on the congratulations text? @rt4914 do you know?

Reply:
It looks like shadow is drawn directly onto the text view [here](https://github.com/oppia/oppia-android/blob/ddbab9626540468343b1b10903f46a969c5c7236/app/src/main/res/layout/state_fragment.xml#L132)

Reply:
Ah, in that case this seems fine to me. Thanks for checking.